### DISCUSSION TITLE
List of container names

### DISCUSSION THREAD

**Discussion by baxterchippy** [Security-Onion-Solutions/securityonion#7332] [Unsupported Versions] [answered]
I was trying to locate the list of containers used in Security Onion? Specifically for 2.3.91?

**Comment by njohnson-q** [comment]
You can use `docker ps` to list out all the containers on each host. You can also save them to a file for easier viewing by using `docker ps > containers.txt` and then using vi or cat to view the file.

**Reply by baxterchippy** [reply]
I know that would show me the "running" containers. How would I get the containers that are on the system but not "running" at that time.

**Reply by tomjones1977** [reply]
`docker ps -a`

**Comment by dougburks** [comment, marked as answer]
To see a list of all container images:
```
sudo docker images
```

This has been added to the documentation:
https://docs.securityonion.net/en/2.3/docker.html#images
